packages/eslint-plugin-plugins/CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to Grafana / ESLint Plugin Plugins
+
+We are always grateful to receive contributions!<br />
+The following guidelines help you on how to start with the codebase and how to submit your work.
+
+## Installation
+
+### Prerequisites
+
+You need to have `npm` installed.
+
+### Installing
+
+```bash
+git clone git@github.com:grafana/plugin-tools.git
+cd plugin-tools
+npm install
+```
+
+## Overview
+
+### Technologies
+
+TBD
+
+### Folder structure
```

Comment:
I would probably do one of two things:

- Remove this part until we have proper content
- Add some proper content here before merging

Reply:
Great catch I'll fix 👍